Jenkinsfile

```diff
@@ -39,14 +39,13 @@ node {
         sh "docker build --tag ${image}:latest --tag ${image}:${tag} ."
       }
       stage ('Test') {
-        withEnv(["COMPOSE_FILE=docker-compose.test.yml"]) {
-          sh "docker-compose up --detach"
-          try {
-              sh "docker-compose exec -T app ./manage.py migrate"
-              sh "docker-compose exec -T app ./manage.py test"
-          } finally {
-              sh "docker-compose down --volumes --remove-orphans"
-          }
+        sh "docker-compose up --detach"
+        try {
+            sh "scripts/ci-setup"
+            // Consider running this through scripts/ci-tests ?
+            sh "docker-compose exec -T app ./manage.py test"
```

Comment:
...change this to `sh "scripts/ci-tests"`?

Reply:
Agreed here, this also test npm though. But its not a big deal